.pylintdict

```diff
@@ -1,7 +1,11 @@
+abelian
+abstractmethod
 adag
 adam
+adjoints
 ae
 aer
+Aer's
```

Comment:
Pylint shouldn't see this as error if `aer` is already in the dict, I think this can be removed.

Reply:
Looking at the travis run you will see the error without Aer's. Running under MacOS without this word flags the same error https://travis-ci.com/github/Qiskit/qiskit-aqua/jobs/297222926#L11188

Reply:
************* Module aqua.operators.expectation_values.aer_pauli_expectation
qiskit/aqua/operators/expectation_values/aer_pauli_expectation.py:28:0: C0402: Wrong spelling of a word 'Aer's' in a docstring:
 An Expectation Value algorithm for using Aer's operator snapshot to
                                          ^^^^^
Did you mean: ''Ar's' or 'Aler's' or 'Amer's' or 'Der's''? (wrong-spelling-in-docstring)